pages/base_page.py

```diff
@@ -10,3 +10,7 @@ def __init__(self, driver: Remote):
         :param driver: Remote.
         """
         self._driver = driver
+
+    def switch_to_ukrainian(self):
+        if self._driver.current_url != "https://auto.ria.com/uk/":
```

Comment:
for checking - find substring like auto.ria.com/uk/ or /uk/

click on button on page for changing language

Reply:
Done